calicoctl/calico_ctl/node.py

```diff
@@ -436,6 +436,10 @@ def _start_node_container_rkt(ip, ip6, node_image, etcd_envs,
     files to mount on the container
     :return: None.
     """
+    if node_image == "calico/node:latest":
```

Comment:
What about `calico/node:vX.X.X`? Don't we still want to pull those from docker?

Will the release script update this to be the correct version on release?